src/shared_utilities/utils.jl

```diff
@@ -85,16 +85,12 @@ end
 
 
 """
-      dss!(Y::ClimaCore.Fields.FieldVector, p::NamedTuple, t::FT)
+      dss!(Y::ClimaCore.Fields.FieldVector, p::NamedTuple, t)
 
 Computes the weighted direct stiffness summation and updates `Y` in place.
 In the case of a column domain, no dss operations are performed.
 """
-function dss!(
-    Y::ClimaCore.Fields.FieldVector{FT},
-    p::NamedTuple,
-    t::FT,
-) where {FT}
+function dss!(Y::ClimaCore.Fields.FieldVector{FT}, p::NamedTuple, t) where {FT}
```

Comment:
this should be fine! but seeing this reminded me of this https://github.com/CliMA/ClimaLSM.jl/issues/356 (which we need to fix anyways, but which made me thing that right now our FieldVectors are not always with parametric type {FT}.

FT can be anything here, so it doesnt affect things, but also FT isnt used in the function. maybe here we can just get rid of it?